data/sdsswork.cfg

```diff
@@ -271,69 +271,70 @@ sdR_N = $BOSS_SPECTRO_DATA_N/{mjd}/sdR-{br}{id}-{frame}.fit
+spAllField_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spectra/@spcoaddfolder|/full/@spcoaddgrp|/{coadd}@spcoaddobs|/{mjd}/spAll-{coadd}@spcoaddobs|-{mjd}.fits
+spAllLineField_coadd = $BOSS_SPECTRO_REDUX/{run2d}/spectra/@spcoaddfolder|/full/@spcoaddgrp|/{coadd}@spcoaddobs|/{mjd}/spAllLine-{coadd}@spcoaddobs|-{mjd}.fits
+
 #boss DRP Epoch Coadd Paths
-sciExp = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/SciExp-@pad_fieldid|.par
-spPlancombepoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/spPlancombepoch-@pad_fieldid|-{mjd}.par
-fieldlist_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/fieldlist-{run2d}.fits
-spAll_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/spAll-{run2d}.fits
-spAll-lite_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/spAll-lite-{run2d}.fits
-spAllLine_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/spAllLine-{run2d}.fits
-spCFrame_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/spCFrame-{br}{id}-{frame:0>8}.fits.gz
-spFluxcalib_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/spFluxcalib-{br}{id}-{frame:0>8}.fits.gz
-spFluxcorr_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/spFluxcorr-{br}{id}-{frame:0>8}.fits.gz
-spFluxdistort_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/spFluxdistort-@pad_fieldid|-{mjd}-{frame:0>8}.fits
-spField_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/spField-@pad_fieldid|-{mjd}.fits
-spZall_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/{run1d}/spZall-@pad_fieldid|-{mjd}.fits
-spZbest_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/{run1d}/spZbest-@pad_fieldid|-{mjd}.fits
-spZline_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/{run1d}/spZline-@pad_fieldid|-{mjd}.fits
-spXCSAO_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/{run1d}/spXCSAO-@pad_fieldid|-{mjd}.fits
-spSpec_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/epoch/coadd/{mjd}/spSpec-@pad_fieldid|-{mjd}-{catalogid}.fits
-specLite_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/spectra/lite/@pad_fieldid|@isplate|/{mjd}/spec-@pad_fieldid|-{mjd}-{catalogid}.fits
-specFull_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/spectra/full/@pad_fieldid|@isplate|/{mjd}/spec-@pad_fieldid|-{mjd}-{catalogid}.fits
-spAllField_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/spectra/full/@pad_fieldid|@isplate|/{mjd}/spAll-@pad_fieldid|-{mjd}.fits
-spAllLineField_epoch = $BOSS_SPECTRO_REDUX/{run2d}/epoch/spectra/full/@pad_fieldid|@isplate|/{mjd}/spAllLine-@pad_fieldid|-{mjd}.fits
+sciExp = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/SciExp-@pad_fieldid|.par
+spPlancombepoch = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/epoch/spPlancombepoch-@pad_fieldid|-{mjd}.par
+fieldlist_epoch = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/fieldlist-{run2d}@epochflag|.fits
```

Comment:
This change is from `/epoch/` to the new `sptypefolder` method, which I think for some cases should return just `epoch` to retain backwards compatibility, but when I look at that method, it returns `summary/epoch` for `fieldlist-epoch`?    I just looked at one example as a check, but want to sure all the bases are covered.

Reply:
You are correct, I missed that when I checked, since it was a valid path to the daily version. I have now fixed the special function